libsignal-service/Cargo.toml

```diff
@@ -12,6 +12,7 @@ async-trait = "0.1.30"
 url = { version = "2.1.1", features = ["serde"] }
 base64 = "0.13"
 bytes = "0.5"
+chrono = "0.4"
```

Comment:
It's a bit sad that we include `chrono` for just `Utc::now().timestamp_millis()`.

Reply:
I added a suggestion below to not have to import `chrono`.